src/mcpbr/benchmarks/agentbench.py

```
@@ -0,0 +1,197 @@
+        instance_id = task.get("instance_id", "agentbench_unknown")
+        environment = task.get("environment", "general")
+        temp_task = {
+            "instance_id": instance_id,
+            "repo": f"agentbench/{environment}",
+            "base_commit": "HEAD",
+        }
+        return await docker_manager.create_environment(temp_task)
+
+    async def evaluate(
+        self,
+        env: TaskEnvironment,
```

Ruff (0.14.14)

[warning] Unused method argument: `env`

(ARG002)